### DISCUSSION TITLE
Auto-create program shortcuts like wine

### DISCUSSION THREAD

**Discussion by upintheairsheep** [darlinghq/darling#1422] [Ideas]
I know that most apps will not work in a long time, but can shortcuts for applications automatically be created similar to Wine?

**Comment by bugaevc** [comment]
Not exactly shortcuts, but it should be possible to create `.desktop` files for the installed apps, so that they will then show up in your DE. See https://github.com/darlinghq/darling/issues/43#issuecomment-419639968 for some ideas.

**Reply by upintheairsheep** [reply]
That is what I am suggesting
